src/index.html

```diff
@@ -8,6 +8,54 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Stars</h1>
+    <div>
+      <div class="row stars--0">
```

Comment:
It should be `stars` block (BEM)
Apply all over the document
```suggestion
      <div class="stars stars--0">
```